src/libs/dataUseTranslation.js

```diff
@@ -241,7 +246,7 @@ export const DataUseTranslation = {
       statementArray = concat(statementArray)(srpTranslations.controls);
     }
 
-    if(darInfo.population || darInfo.poa) {
```

Comment:
`population` is still a valid DAR term that exists on current production DARs (i.e. see "5bad3b6d46e0fb0001f2ba7b") so we still need to support it.